projectroles/tests/test_commands.py

```diff
@@ -495,6 +501,18 @@ def _make_undefined_setting(self):
         ghost.save()
         return ghost
 
+    def _make_setting_category_type(self):
```

Comment:
If this is only used once in one test, we probably don't have to add a separate method for it. Instead, it can just be created at the beginning of the test.